### PR TITLE
docs: update changelog for 3.0.0

### PR DESCRIPTION
**Pull request by istankovic**

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,15 @@
 Changelog
 =========
 
+* :release:`2.1.0 <2021-08-25>`
+* :feature:`7163` Add a Matrix federation checker tool.
+* :feature:`7162` Don't prune routes anymore.
+* :feature:`7196` Send encrypted secret with the LockedTransfer message, obviating the need for SecretRequest and SecretReveal messages. This reduces the time needed to complete a payment.
+* :feature:`7195`, `7247` Enable WebRTC support. We now keep exactly one open WebRTC channel per peer, and regularly check for channel availability.
+* :feature:`7233`, `7264` Add support for cooperative settlement. If both parties are online and able to settle cooperatively, settlement is performed in a much quicker fashion. Closing a channel will now automatically try doing cooperative settlement first.
+* :feature:`7276` We don't pin Synapse to 1.33.0 anymore.
```
Comment (ezdac) [inline docs/changelog.rst]: `Remove pin Synapse to 1.33.0.` or similar

File: docs/changelog.rst [hunk 2]
```diff
@@ -2,6 +2,15 @@
 Changelog
 =========
 
+* :release:`2.1.0 <2021-08-25>`
+* :feature:`7163` Add a Matrix federation checker tool.
+* :feature:`7162` Don't prune routes anymore.
+* :feature:`7196` Send encrypted secret with the LockedTransfer message, obviating the need for SecretRequest and SecretReveal messages. This reduces the time needed to complete a payment.
+* :feature:`7195`, `7247` Enable WebRTC support. We now keep exactly one open WebRTC channel per peer, and regularly check for channel availability.
```
Comment (ezdac) [inline docs/changelog.rst]: > We now 

I don't like the we in the changelog

File: docs/changelog.rst [hunk 3]
```diff
@@ -2,6 +2,15 @@
 Changelog
 =========
 
+* :release:`2.1.0 <2021-08-25>`
+* :feature:`7163` Add a Matrix federation checker tool.
+* :feature:`7162` Don't prune routes anymore.
```
Comment (ezdac) [inline docs/changelog.rst]: Remove pruning for routes in LockedTransfer metadata.

**Review by ezdac — CHANGES_REQUESTED** [review]
Thanks for gathering all the issues!
I added some writing-style comments

File: docs/changelog.rst (hunk 1 above)
Comment (istankovic) [reply docs/changelog.rst]: How about "Synapse is not pinned to 1.33.0 anymore"?

**Pull request merged**